src/mapping/sanity/expert-advice.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button } from '@src/components/features/decoupled-components/button/button';
```

Comment:
In the mapping folder shouldn't be any component code, all data/query related code should go in folder data, all component code should be in folder component, the only thing that the mapping is supposed to do is to connect the sanity specific data to the generic component data it belongs to, it transfers (or slightly alters) the data wherever needed. Can you please split the code into the right parts? Thanks. Feel free to ask questions if needed ;-)